tests/test_screenshot.py

```diff
@@ -99,3 +99,58 @@ async def test_screenshot(websocket, context_id, png_base64):
     img1 = Image.open(io.BytesIO(base64.b64decode(resp["result"]["data"])))
     img2 = Image.open(io.BytesIO(base64.b64decode(png_base64)))
     assert_images_equal(img1, img2)
+
+
+@pytest.mark.asyncio
+async def test_screenshot_oopif(websocket, context_id, html, iframe):
+    await goto_url(websocket,
+                   context_id,
+                   html(iframe("https://www.example.com")),
```

Comment:
Could you please add a small margin for the iframe, so that we are sure it is not defaults to (0,0)?

Reply:
Actually, this is not necessary.

It turns out fetching example.com doesn't work from github actions (see E2E tests failing in CI, although they do pass locally).

So we cannot add an OOPIF test at all.

Reply:
It's hard to understand the reason. Maybe save the images to artefacts when a test fails?

Reply:
Skipping this for now, in order to submit the change that uses the pngs instead of base64